pom.xml

```diff
@@ -18,321 +18,328 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <parent>
-    <groupId>org.jenkins-ci</groupId>
-    <artifactId>jenkins</artifactId>
-    <version>1.32</version>
-  </parent>
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>org.jenkins-ci.lib</groupId>
-  <artifactId>lib-jenkins-maven-embedder</artifactId>
-  <name>Jenkins Maven Embedder</name>
-  <version>3.12-SNAPSHOT</version>
-  
-  <prerequisites>
-    <maven>3.0</maven>
-  </prerequisites>
-   
-  <scm>
-    <connection>scm:git:git://github.com/jenkinsci/lib-jenkins-maven-embedder.git</connection>
-    <developerConnection>scm:git:ssh://git@github.com/jenkinsci/lib-jenkins-maven-embedder.git</developerConnection>
-    <url>https://github.com/jenkinsci/lib-jenkins-maven-embedder</url>
-    <tag>HEAD</tag>
-  </scm>
-  
-  <properties>
-    <aetherVersion>0.9.0.M2</aetherVersion>
-    <mavenVersion>3.1.0</mavenVersion>
-    <wagonVersion>2.4</wagonVersion>
-    <sisuInjectVersion>0.0.0.M2a</sisuInjectVersion>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <slf4jVersion>1.7.4</slf4jVersion>
-  </properties>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
```

Comment:
Is the wholesale reformatting really necessary?  It makes reading the diffs more difficult (though I guess I can add the ?w=1 argument to the URL)